Makefile

```diff
@@ -5,7 +5,7 @@ K := $(foreach exec,$(EXECUTABLES),\
 ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
 PACKAGE_NAME?=github.com/packethost/packet-cli
```

Comment:
Perhaps we should replace this with github.com/equinix/metal-cli in a separate commit (and capture that change in go.mod too).

Reply:
It looks like we can replace `packethost/packet-cli` with `equinix/metal-cli` everywhere.
And then another replace to get `packet-cli` converted to `metal-cli` will pick up some stragglers.

Reply:
Agreed. I wasn't sure if this was appropriate at this stage, but if it won't be a problem, then will do.